src/main/resources/application.properties

```diff
@@ -98,6 +98,8 @@ rundeck.project.group.quickstarter=quickstarts
 rundeck.project.group.openshift=openshift
 rundeck.project.openshift.create.name=create-projects
 
+jenkinspipeline.quickstarter={beSpringBoot: 'http://192.168.56.31:7990/scm/prov/quickstarter-jenkins.git#master/beSpringBoot/Jenkinsfile'}
```

Comment:
why do we have to encode an IP here .. we have bitbucket host etc already in the property file?!

Reply:
The default should be the Bitbucket repo that is alread defined ...

Reply:
As stated in https://github.com/opendevstack/ods-provisioning-app/issues/270, I want the whole host to go away - we don't need it.